cypress/integration/dictionary/addBulkConceptsToDictionary.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Add bulk concepts to an existing dictionary
+  Background:
+    Given the user is logged in
+
+  @dictionary
+  Scenario: The user should be able to add bulk concepts from their preferred source
+    Given the user is on the view dictionary concepts page
+     When the user clicks the "Add concepts" button
```

Comment:
We now have two steps called "the user clicks the "Add concepts" button" trying to test very different things (one is clicking the "Add concepts" button on the bulk import page. One is clicking the little "+" icon on the main dictionary view.

We need to describe these steps differently so we can have different implementations for them.

Reply:
@ibacher thanks so so much I deleted  the little "+" icon on the main thinking I have repeated the implementation. Thanks so so much for helping me spot my endless errors.

Reply:
We still need to be able to distinguish between "the user clicks the "Add concepts" button" here and the other one we use. The element that we display the "+" with is called a ["floating action button"](https://material.io/components/buttons-floating-action-button) in the Material Design framework because it's designed to remain on the user's screen regardless of scroll position, height, etc. I.e., it's "floating" there in the lower-right.

So let's name this step:

```
When the user clicks on the "Add concepts" floating action button
```

And leave the other text for the other button with an "Add concepts" label.

Reply:
@ibacher am sorry I have just seen the above comment but I have fixed it.